cipher/errors.go

```diff
@@ -0,0 +1,10 @@
+package cipher
+
+import "errors"
+
+// ErrorFailedToEncrypt Raised when a cipher function fails to encrypt the message
+var ErrorFailedToEncrypt = errors.New("failed to Encrypt")
```

Comment:
```suggestion
var ErrorEncrypt = errors.New("cipher: failed to encrypt")
```
Maybe even change this to a struct with the necessary data instead of a simple string.